src/backends/cmake.rs

```diff
@@ -43,37 +39,31 @@ impl Backend for Cmake {
         cmake_command.arg(format!("{}/src-gen", self.lfc.out.display().to_string()));
         cmake_command.arg(format!("-B {}/build", self.lfc.out.display().to_string()));
         cmake_command.current_dir(format!("{}/build", self.lfc.out.display().to_string()));
-        let cmake_gen = run_and_capture(&mut cmake_command).is_ok();
+        run_and_capture(&mut cmake_command)?;
 
         // compiling
         let mut cmake_build_command = Command::new("cmake");
         cmake_build_command.current_dir(format!("{}/build", self.lfc.out.display().to_string()));
         cmake_build_command.arg("--build");
         cmake_build_command.arg("./");
-        let cmake_build = run_and_capture(&mut cmake_build_command).is_ok();
+        run_and_capture(&mut cmake_build_command)?;
```

Comment:
This is how you use result: if the function `run_and_capture` returns an error result, then the question mark operator here causes the current function to return the same error result. That way in the rest of this function you can assume that the call succeeded.